### DISCUSSION TITLE
Getting started with Raspberry Pi and Tang Nano 9k

### DISCUSSION THREAD

**Discussion by Totalis** [multigcs/LinuxCNC-RIO#45] [Q&A]
Hi All,

I am trying to get a basic example of LinuxCNC-RIO working on my Raspberry Pi 4B (Debian Bookworm, Linuxcnc 2.9.1) using a Tang Nano.  I have got the Pi to boot and linuxcnc is installed, and I also have the Tang Nano to hand:

1. How do I install the Linuxcnc component that can talk to the Tang Nano?
2. How do I generate and install the bitfile for the Tang Nano?

Many thanks, its awesome to see such development in the LinuxCNC ecosystem

Greg

**Comment by multigcs** [comment]
You can try this install-doc:
https://github.com/multigcs/LinuxCNC-RIO/blob/main/configs/TangNano9K/INSTALL.md

but you need another linux-system (maybe VM) to build and flash the tang-nano, the FTDI-Fake chip on this board is very buggy and i have problems to flash it using a raspberry-pi :(
